openpype/plugins/publish/collect_context_label.py

```diff
@@ -16,16 +17,22 @@ class CollectContextLabel(pyblish.api.ContextPlugin):
     label = "Context Label"
 
     def process(self, context):
-
-        # Get last registered host
-        host = pyblish.api.registered_hosts()[-1]
-
-        # Get scene name from "currentFile"
-        path = context.data.get("currentFile") or "<Unsaved>"
-        base = os.path.basename(path)
+        # Add ability to use custom context label
+        context_label = context.data.get("contextLabel")
+        if context_label:
+            context.data["label"] = context_label
+            return
```

Comment:
If this is the case - why not instead have it do `if context.data.get("label")` and ignore this plug-in if a label was already set for the context. Or is `context.data["label"]` set usually?

Reply:
I think that `context.data["label"]` is set to `Context` by default.

Reply:
Did some research and found out it's used that way in pyblish pype.

Reply:
Changed